pypit/ginga.py

```diff
@@ -84,7 +85,15 @@ def show_image(inp, chname='Image', **kwargs):
     viewer = connect_to_ginga()
     ch = viewer.channel(chname)
     name='image'
-    ch.load_np(name, img, 'fits', {})
+    # Header
+    header = {}
+    header['NAXIS1'] = img.shape[1]
+    header['NAXIS2'] = img.shape[0]
+    header['WCS-xIMG'] = 'MF_lris_red/MasterWave_A_02_aa.fits'
+    import pdb; pdb.set_trace()
```

Comment:
Also, if we import at the top of the file, do we need to import again here? If it's just so that the user can pause, we can use a `msgs.input()` or, if you want it outside of PYPIT, you could just use a `raw_input()`.